storey/steps/flatten.py

```diff
@@ -1,18 +1,5 @@
-from storey import Flow
-from storey.dtypes import _termination_obj
+from storey import FlatMap
 
 
-class Flatten(Flow):
-    """
-    Splits an event with an iterable body into multiple events.
-    """
-
-    def __init__(self, **kwargs):
-        super().__init__(full_event=True, **kwargs)
-
-    async def _do(self, event):
-        if event is _termination_obj:
-            return await self._do_downstream(_termination_obj)
-        else:
-            for element in event.body:
-                await self._do_downstream(event.copy(body=element))
+def Flatten(**kwargs):
+    return FlatMap(lambda x: x, **kwargs)
```

Comment:
The only thing is, I think passing `full_event=True` will always break. Although this is probably not so important.